manual/python/index.md

```diff
@@ -555,8 +570,8 @@ The purpose of these classes was to serve as a base class for Python classes tha
 from the ROOT::Math classes. This allowed to define Python functions that could be used for fitting
 in Fit::Fitter.<br/>
 In the new PyROOT, `TPyMultiGenFunction` and `TPyMultiGradFunction` do not exist anymore, since their
-functionality is automatically provided by new cppyy: when a Python class inherits from a C++ class,
-a wrapper C++ class is automatically generated. That wrapper class will redirect any call from C++
+functionality is automatically provided by new cppyy. When a Python class inherits from a C++ class,
```

Comment:
With the `:` here I meant to explain how the functionality is automatically provided by new cppyy.

Reply:
sure, but then it a very long sentence. But anyhow "when" must be reflaced by "if".